### DISCUSSION TITLE
What is TipSetKey ?

### DISCUSSION THREAD

**Discussion by s1mple1122** [filecoin-project/lotus#4837] [Storage Provider] [answered]
Tell me the format or doc please ,I really can't understand your notes;

**Comment by s1mple1122** [comment]
request APi ,if need TipSetKey, i always dont;

**Comment by jennijuju** [comment, marked as answer]
List of block cids. Or you can pass null.

**Comment by jennijuju** [comment]
Not an issue but a valid question for discussion.
